WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift

```diff
@@ -141,26 +141,27 @@ final class OrderPaymentDetailsViewModelTests: XCTestCase {
         XCTAssertNotNil(viewModel.paymentSummary)
     }
 
-    func test_payment_summary_contains_awaiting_payment_message_when_date_paid_is_null() {
-        let awaitingPayment = String.localizedStringWithFormat(
-            NSLocalizedString(
-                "Awaiting payment via %@",
-                comment: "A unit test string. It reads: " +
-                "Awaiting payment via <payment method title>." +
-                "A payment method example is: 'Credit Card (Stripe)'"
-            ),
-            anotherBrokenOrder.paymentMethodTitle
-        )
-
-        XCTAssertEqual(anotherBrokenOrder.paymentMethodTitle, "Cash on Delivery")
-        XCTAssertNil(anotherBrokenOrder.datePaid)
-
-        guard let paymentSummary = anotherBrokenOrderViewModel.paymentSummary else {
-            XCTFail("The payment summary should not be nil or blank.")
+    func test_payment_summary_returns_nil_when_date_paid_is_nil() {
+        guard let _ = order.datePaid else {
+            XCTAssertNil(anotherBrokenOrderViewModel.paymentSummary)
             return
         }
```

Comment:
`guard let` shouldn't be necessary in this case since we don't need to return early from any logic here. We can simply assert that `datePaid` is nil for the confidence check:
```suggestion
        XCTAssertNil(order.datePaid) // confidence check
        XCTAssertNil(anotherBrokenOrderViewModel.paymentSummary)
```

Reply:
That makes sense! I've split this into two different tests for when `datePaid` nil and not-nil, as now we will return "Awaiting payment" for the `datePaid == nil` case